docs/src/shared/components/InfoCard/InfoCard.tsx

```diff
@@ -97,7 +97,9 @@ export const InfoCard: React.FC<InfoCardProps> = (props): JSX.Element => {
                     </div>
                 </Box>
             )}
-            <Typography variant={'h6'}>{props.title}</Typography>
+            <Typography variant={'h6'} color={'text.primary'}>
+                {props.title}
+            </Typography>
             <Typography variant={'body2'} style={{ color: theme.palette.text.secondary, marginTop: theme.spacing(1) }}>
```

Comment:
I missed this while migrating from doc-it. Can you please update this to have sx prop:
`sx={{color: 'text.secondary', mt: 1}}`